src/utils.js

```diff
@@ -21,13 +21,11 @@ export function formatISOString(isoString, format = 'DD.MM.YYYY') {
 }
 
 export function isValidUrl(input) {
-    const pattern = new RegExp('^(https?:\\/\\/)' // protocol (requires http://or https://)
-        + '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,})' // domain name
-        + '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' // port and path
-        + '(\\?[;&a-z\\d%_.~+=-]*)?' // query string
-        + '(\\#[-a-z\\d_]*)?$', 'i'); // fragment locater
-
-    return pattern.test(input);
+    try {
+        return new URL(input).protocol.startsWith('http');
```

Comment:
Er det innafor?

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#Browser_compatibility

Hvis IE 11 ikke trenger å støttes, så er det kjempemye mer leselig :-)

Reply:
Ikke sikker. Dvs. jeg er sikker på at metoden er overlegen, men det avhenger selvfølgelig av browser-kompat. For meg så det ut til å ha vært støttet en god stund i browsere, men jeg sjekket ikke IE11.

Reply:
Jeg lager en graceful-degrade på det. Hvis IE11, så vil alltid denne funksjonen returnere `false`, og søknadslenker på stillinger kun vises, men ikke være klikkbare. Synes det er et greit kompromiss.

Reply:
Enig i at denne metoden er overlegen

Hvor mange av brukerene våre er på IE11? Hvis det er en svært liten del, så er jeg enig i at det er et godt kompromiss.

Reply:
Jeg la på en enkel sjekk som ikke utfører valideringen hvis MSIE (da validerer ingen URL-er, men eneste konsekvens er at de ikke blir lenket opp, så det går fint). Testet i IE11 OK, regresjonstestet i Firefox og Chrome.